docs/subsystems/hashchange-system.md

```diff
@@ -8,9 +8,9 @@ be used to deep-link into the application and allow the browser's
 Some examples are:
 
 - `/#settings/your-bots`: Bots section of the settings overlay.
-- `/#streams`: Streams overlay, where the user manages streams
+- `/#channels`: Streams overlay, where the user manages streams
```

Comment:
There's more content updates to be done on this page, but felt that it would be better done in a focused sweep for the stream to channel rename in `docs/`

Reply:
Yep sgtm!